resume/src/App.css

```diff
@@ -0,0 +1,43 @@
+.App {
+  text-align: center;
```

Comment:
It's probably better to use `margin: 0 auto` if you want to be centring your entire `.App`, as this will apply to the block elements opposed to just the text.